xbmc/cores/VideoPlayer/DVDCodecs/Video/AddonVideoCodec.cpp

```diff
@@ -199,7 +199,7 @@ CDVDVideoCodec::VCReturn CAddonVideoCodec::GetPicture(VideoPicture* pVideoPictur
     pVideoPicture->pts = static_cast<double>(picture.pts);
     pVideoPicture->dts = DVD_NOPTS_VALUE;
     pVideoPicture->color_range = 0;
-    pVideoPicture->color_matrix = 4;
+    pVideoPicture->color_space = 4;
```

Comment:
@peak3d why do you set color space to 4, which is: FCC Title 47 Code of Federal Regulations 73.682
does not make much sense to me

Reply:
@FernetMenta cp/paste from some source, no idea what I do here, so pls. let me know how to evaluate this value.

Reply:
B.t.w. same code is in AML

Reply:
@peak3d you should detect color space of the source and set value according to AVColorSpace (ffmpeg). If source has no color space set, set it to AVCOL_SPC_UNSPECIFIED